### ISSUE TITLE
[Dynamo] InternalTorchDynamoError: __init__() got an unexpected keyword argument '_saved_tensors'

### ISSUE THREAD

**Issue by yanboliang** [pytorch/pytorch#103460]
### 🐛 Describe the bug

Check how to setup 14k github models at [here](https://github.com/pytorch/pytorch/issues/93667#issue-1565238954).

Repro:
```
pytest ./generated/test_pabloppp_pytorch_tools.py -k test_015
```

Error:
```
  File "/scratch/ybliang/work/repos/pytorch/torch/_dynamo/variables/misc.py", line 391, in __init__
    super().__init__(value=value, value_type=value_type, **kwargs)
  File "/scratch/ybliang/work/repos/pytorch/torch/_dynamo/variables/user_defined.py", line 173, in __init__
    super().__init__(**kwargs)
torch._dynamo.exc.InternalTorchDynamoError: __init__() got an unexpected keyword argument '_saved_tensors'

from user code:
   File "./generated/test_pabloppp_pytorch_tools.py", line 677, in forward
    ctx.mark_non_differentiable(indices)
```

### Versions

N/A

cc @ezyang @msaroufim @wconstab @bdhirsh @anijain2305 @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @Xia-Weiwen @wenzhe-nrv @jiayisunx @ipiszy

**Comment by yanboliang** [comment]
This is an allocated bootcamp task, assigned to me until the bootcamper claim it.

**Comment by yanboliang** [comment]
resolved by #103594
